ElenHakobjanyan/Words Frequencies

```diff
@@ -0,0 +1,86 @@
+#include <iostream>
+#include <iomanip>
+#include <fstream>
+#include <string>
+#include <map>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
+
+bool addWordsToMap(string, map<string, int>&, int&);
+void sortAsVec(map<string, int>&, vector<pair<string, int>>&);
+void print(vector<pair<string, int>>&, int, int&);
+
+int main()
+{
+	int count = 0;
```

Comment:
what is `count` for? Is that just the number of words that is the same as map size?

Reply:
it's the count of all words, for %,      map size will be only not repeating ones

Reply:
i see

Reply:
for count i can also cycle over the map once(in the print function) and add all values to count, it must give the same amount, but would it be better?

Reply:
Not a big deal. Either way